packages/jsii-reflect/lib/docs.ts

```diff
@@ -29,6 +32,15 @@ export class Docs {
     return ret.join('\n');
   }
 
+  /**
+   * Returns whether or not the stability should be mentioned
+   */
+  public shouldMentionStability(): boolean {
```

Comment:
NIT: I think the naming here implies a rule that the logic itself doesn't necessarily mean. Whether a target includes stability only if not stable seems language specific. Something like "isUnstable" seems more accurate.

This also can be a getter instead of a method.

Reply:
This was ported from both the [python](https://github.com/aws/jsii/pull/1976) and [dotnet](https://github.com/aws/jsii/pull/1976) generators. It seemed like something that could be DRY'd up, so if you prefer I can just put it in the golang generator.

Reply:
I do think we should move to the golang generator in that case just it's my feeling that the reflect library doesn't own this and each language implementation does.

Reply:
I can do this, but just FYI the reason why I put this in the reflect library is, as I mentioned offline and earlier, not everything in the underlying `jsii` lib is in the reflect library and I was trying to avoid having to import `@jsii/spec`. Since the python and dotnet generators both import the spec library instead of the reflect library, they have access to the Stability types, which currently does not exist in the reflect lib (See: https://github.com/aws/jsii/blob/master/packages/jsii-pacmak/lib/targets/dotnet/dotnetdocgenerator.ts#L217).